src/components/query-viewer-pivot-table/query-viewer-pivot.tsx

```diff
@@ -66,6 +70,33 @@ export class QueryViewerPivot {
     setDataSynForPivotTable(this.qViewer.oat_element, newValue);
   }
 
+  /**
+   * Response Page Data
+   */
+  @Prop() readonly pageDataForTable: string;
+  @Watch("pageDataForTable")
+  handlePageDataForTableChange(newValue: string) {
+    setPageDataForTable(this.qViewer.oat_element, newValue);
+  }
+
+  /**
+   * Response Attribute Values
+   */
+  @Prop() readonly attributeValuesForTableXml: string;
+  @Watch("attributeValuesForTableXml")
+  handleAttributesValuesForTableChange(newValue: string) {
+    setAttributeForTable(this.qViewer.oat_element, newValue);
+  }
+
+  /**
+   * Response Table Data Sync
+   */
+  @Prop() readonly getTableDataSyncXml: string;
```

Comment:
It sound like a method instead of a property. Can it be `tableDataSyncXml`?
```suggestion
  @Prop() readonly tableDataSyncXml: string;
```